java/code/webapp/WEB-INF/nav/system_detail.xml

```diff
@@ -210,7 +220,7 @@
   </rhn-tab>
 
   <rhn-tab name="Ansible" acl="system_has_ansible_control_node_entitlement()">
-    <rhn-tab-url>/rhn/manager/systems/details/ansible/control-node</rhn-tab-url>
+      <rhn-tab-url>/rhn/manager/systems/details/ansible/control-node</rhn-tab-url>
```

Comment:
nitpick: indentation error

Reply:
Indentation of this file is a bit of. In theory it's fixed now.